Easydict/Swift/Utility/RepoInfoHelper.swift

```diff
@@ -0,0 +1,218 @@
+    var htmlUrl: String?
+    var id: Int = 0
+    var login: String?
+    var nodeId: String?
+    var organizationsUrl: String?
+    var receivedEventsUrl: String?
+    var reposUrl: String?
+    var siteAdmin: Bool = false
+    var starredUrl: String?
+    var subscriptionsUrl: String?
+    var type: String?
+    var url: String?
+}
+
+// MARK: - RepoInfoResponseReactions
+
+struct RepoInfoResponseReactions: Codable {
+    enum CodingKeys: String, CodingKey {
+        case reactionsAdd = "+1"
+        case reactionsSub = "-1"
+        case totalCount = "total_count"
+    }
+
+    var reactionsSub: Int = 0
+    var reactionsAdd: Int = 0
+    var confused: Int = 0
+    var eyes: Int = 0
+    var heart: Int = 0
+    var hooray: Int = 0
+    var laugh: Int = 0
+    var rocket: Int = 0
+    var totalCount: Int = 0
+    var url: String?
+}
+
+// MARK: - RepoInfoResponseAuthor
+
+struct RepoInfoResponseAuthor: Codable {
+    enum CodingKeys: String, CodingKey {
+        case avatarUrl = "avatar_url"
+        case eventsUrl = "events_url"
+        case followersUrl = "followers_url"
+        case followingUrl = "following_url"
+        case gistsUrl = "gists_url"
+        case gravatarId = "gravatar_id"
+        case htmlUrl = "html_url"
+        case nodeId = "node_id"
+        case organizationsUrl = "organizations_url"
+        case receivedEventsUrl = "received_events_url"
+        case reposUrl = "repos_url"
+        case siteAdmin = "site_admin"
+        case starredUrl = "starred_url"
+        case subscriptionsUrl = "subscriptions_url"
+    }
+
+    var avatarUrl: String?
+    var eventsUrl: String?
+    var followersUrl: String?
+    var followingUrl: String?
+    var gistsUrl: String?
+    var gravatarId: String?
+    var htmlUrl: String?
+    var id: Int = 0
+    var login: String?
+    var nodeId: String?
+    var organizationsUrl: String?
+    var receivedEventsUrl: String?
+    var reposUrl: String?
+    var siteAdmin: Bool = false
+    var starredUrl: String?
+    var subscriptionsUrl: String?
+    var type: String?
+    var url: String?
+}
+
+// MARK: - RepoInfoHelper
+
+class RepoInfoHelper {
+    // MARK: Lifecycle
+
+    private init() {
+        //
+    }
+
+    // MARK: Public
+
+    public func fetchLatestVersion(repoPath: String, completion: @escaping (String) -> ()) {
+        fetchRepoInfo(repoPath: repoPath) { response in
+            switch response {
+            case let .success(resp):
+                completion(resp.tagName ?? "")
+            case let .failure(err):
+                print(err)
+            }
+        }
+    }
+
+    // MARK: Internal
+
+    static let shared = RepoInfoHelper()
+
+    // MARK: Private
+
+    private func fetchRepoInfo(repoPath: String, completion: @escaping (Result<RepoInfoResponse, Error>) -> ()) {
+        let url = "https://api.github.com/repos/\(repoPath)/releases/latest"
+        AF.request(url).responseDecodable(of: RepoInfoResponse.self) { response in
```

Comment:
Can we use AF `async` instead of completion?  It's more concise and readable.

[Using Alamofire with Swift Concurrency](https://github.com/Alamofire/Alamofire/blob/master/Documentation/AdvancedUsage.md#using-alamofire-with-swift-concurrency)

Reply:
okay I will change  it tomorrow

Reply:
I have done 13189687